scripts/simulate

```diff
@@ -12,6 +13,7 @@ do
     case "$1"
     in
     --elf)                   elf="$2";   shift 2;;
+    --spike)               spike="$2";   shift 2;;
```

Comment:
Nick expressed that he also need to specify spike options/configs just like qemu.
Is that a following PR by you or Nick?

Reply:
My hope is that `SPIKE_OPTIONS` environment variable will (a) allow `freedom-e-sdk` to discern the appropriate Device Tree and build itself appropriately (b) get passed to `spike` as command-line options. It looks to me like the current PR includes this functionality.